scripts/test.sh

```diff
@@ -425,11 +425,46 @@ function unparam_pass {
   run_for_modules generic_checker run_go_tool "mvdan.cc/unparam"
 }
 
-function staticcheck_pass {
-  # TODO: we should upgrade pb or ignore the pb package
-  #
-  # versionpb/version.pb.go:69:15: proto.RegisterFile is deprecated: Use protoregistry.GlobalFiles.RegisterFile instead.  (SA1019)
-  run_for_modules generic_checker run_go_tool "honnef.co/go/tools/cmd/staticcheck"
+function golangcilint_pass {
+  local tmp_lintyaml
+  tmp_lintyaml=$(mktemp -t 'tmp_golangcilint_XXX.yaml')
+
+  # shellcheck disable=SC2064
+  trap "rm ${tmp_lintyaml}; trap - RETURN" RETURN
+
+  # TODO: Copy the tools/.golangci.yaml and run the linters for which we have
```

Comment:
We should only have 1 source of lint configuration. Adding second one like this is bad for visibility.

Please just disable no working linters in the main golangci.yaml and add a TODO to reenable them.

Reply:
Makes senses. Updated.